terraform/logs.tf

```diff
@@ -1,8 +1,5 @@
 resource "aws_cloudwatch_log_group" "log_group" {
   name              = "/${var.env}/mca/beacons"
+  tags              = module.beacons_label.tags
   retention_in_days = 30
-
-  tags = {
-    Name = "${var.env}-mca-log-group"
-  }
 }
```

Comment:
Considering that suffixing the name with `-mca-log-group` doesn't add useful information.  The type of resource is metadata already provided by AWS.

Reply:
Happy with this. The log group suffix is implied by the resource.  I think it was the `mca` value we were interested in having in but that's now covered off by the tags, so happy to remove